.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,21 @@
 
 - ...
 
-## PR checklist for schema changes
+## Checklist
 
-<!-- If the current schema version already ends in "-alpha", then delete the first two items: -->
-- [ ] Update the version string in `docs/format/source/conf.py`, `core/nwb.namespace.yaml`, and `/core/nwb.file.yaml`
-  to the next version with the suffix "-alpha"
-- [ ] Add a new section in the release notes for the new version with the date "Upcoming"
-- [ ] Add release notes for the PR to `docs/format/source/format_release_notes.rst`
+If this is the first change after a schema release (i.e., the version string in `core/nwb.namespace.yaml` does not end
+in "-alpha"), then:
+- [ ] Update the version string in `core/nwb.namespace.yaml` and `core/nwb.file.yaml` to the next major/minor/patch
+  version with the suffix "-alpha". For example, if the current version is 2.4.0 and this is a minor change, then the
+  new version string should be "2.5.0-alpha".
+- [ ] Update the value of the `version` variable in `docs/format/source/conf.py` to the next version **without** the
+  suffix "-alpha", e.g., "2.5.0".
+- [ ] Update the value of the `release` variable in `docs/format/source/conf.py` to the next version **with** the suffix
+  "-alpha", e.g., "2.5.0-alpha".
+- [ ] Add a new section in the release notes `docs/format/source/format_release_notes.rst` for the new version
+  with the date "Upcoming".
+
+For all schema changes:
+- [ ] Add release notes for the PR to `docs/format/source/format_release_notes.rst`.
```

Comment:
The text looks good.

One minor suggestion. I think it would be slightly more intuitive to move this last item to the beginning of the list. I.e., I would lead with the items that should be done on all PRs and then have the items that are only required on some PRs (e.g., PRs that require updating the version).